monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1397 @@
+            [
+                RowPanel(
+                    title="Overview",
+                    repeat=Repeat(variable="service_name", direction="h"),
+                ),
+                # Overview section with custom layout
+                layout.row(
+                    [
+                        # Left side - MongoDB services state (3 width, 8 height)
+                        *layout.resize([mongodb_services_state], width=3),
+                        # Right side - version and oplog stacked (3 width total)
+                        layout.column(
+                            [
+                                *layout.resize([version_panel], height=3),
+                                *layout.resize([smallest_oplog], height=5),
+                            ],
+                            width=3,
+                            height=8,
+                        ),
+                        # Disk usage pie (4 width, 8 height)
+                        *layout.resize([disk_usage_pie], width=4),
+                        # Activity indicators stacked (4 width, 4 height each)
+                        layout.column(
+                            [
+                                *layout.resize([current_operations], height=4),
+                                *layout.resize([total_active_cursors], height=4),
+                            ],
+                            width=4,
+                            height=8,
+                        ),
+                        # Avg doc size + disk space util stacked (4 width, 4 height each)
+                        layout.column(
+                            [
+                                *layout.resize([avg_doc_size], height=4),
+                                *layout.resize([disk_space_util], height=4),
+                            ],
+                            width=4,
+                            height=8,
+                        ),
+                        # Size + num collections stacked (3 width, 4 height each)
+                        layout.column(
+                            [
+                                *layout.resize([size_collections], height=4),
+                                *layout.resize([num_collections], height=4),
+                            ],
+                            width=3,
+                            height=8,
+                        ),
+                        # Total indexes (3 width, 8 height)
+                        *layout.resize([num_indexes], width=3),
+                    ],
+                    height=8,
+                ),
+                # Config server and shard distribution
+                layout.row(
+                    [
+                        # Left side - Config server data stacked (6 width)
+                        layout.column(
+                            [
+                                *layout.resize([docs_configsvr], height=4),
+                                *layout.resize([index_size_configsvr], height=4),
```

Comment:
resize should not be needed, `layout.column` should do it automatically (I think)